Adding support of .Net Standard 2.0

TdMxm opened the pull request:
Solving https://github.com/ademchenko/DependenciesTracker/issues/26
Adding .Net Standard 2.0 and .Net Standard 1.0 support, so library can be used in .Net Core and modern Mono projects

- migration to Microsoft.SDK style csproj
- builds now targeting net40, net 45, netstandard1.0, netstandard2.0
- tests project now targeting net40

According to https://docs.microsoft.com/ru-ru/dotnet/standard/net-standard .Net Standard 1.0 support fully covers previous Profile78 support

Commit:
https://github.com/ademchenko/DependenciesTracker/issues/26 .Net Standard 2.0 support

- migration to Microsoft.SDK style csproj
- builds now targeting net40, net 45, netstandard1.0, netstandard2.0

ademchenko commented:
@TdMxm, thank you that you found a time to contribute to the project. I will review and get back with the results.

Athari commented:
> tests project now targeting net40

Test projects can have multiple target frameworks too.

Commit:
https://github.com/ademchenko/DependenciesTracker/issues/26 .Net Standard 2.0 support

- tests now targeting net40;net45;netcoreapp3.1
- migrating tests from using PropertyData to ClassData for supporting both xUnit 1.9.2(net4.0) and xUnit 2.2+(.net Core)

TdMxm commented:
> Test projects can have multiple target frameworks too.
fixed
